NewEvaluation.css

```diff
@@ -33,6 +33,9 @@ body{
 .right-button{
 	float:right;
 }
+ul{
```

Comment:
I'd recommend getting into the habit of properly formatting your code - an especially useful habit for front-end developers, since it trains you to instinctively pay attention to detail.